src/test/autodjprocessor_test.cpp

```diff
@@ -22,8 +22,7 @@ using ::testing::Return;
 
 static int kDefaultTransitionTime = 10;
 const mixxx::audio::ChannelCount kChannelCount = mixxx::kEngineChannelCount;
-const QString kTrackLocationTest(QDir::currentPath() %
-                                 "/src/test/id3-test-data/cover-test-png.mp3");
+const QString kTrackLocationTest = QStringLiteral("/id3-test-data/cover-test-png.mp3");
```

Comment:
The absolute paths in these constants are confusing. They should become relative paths that are passed to `QDir::filePath()`.

Reply:
QString QDir::filePath(const &fileName);
Takes a file name only, is not designed to concatenate paths. 

I will remove the leading "/" from this constants instead to avoid the confusion.

Reply:
Wrong. The official documentation states that "Redundant multiple separators or "." and ".." directories in fileName are not removed", i.e. assuming that `fileName` is a (relative) path and not a single file name.

Reply:
They document the side effect, that happens if you use the function with a path instead if a fileName, because all other functions are expecting native seperators.

For instance if QDir::relativeFilePath() expects that the argument has native seperators. While QDir::filePath() literally copies the file name including all slashes. 

But that question of feature or side effect is not relevant here.

It is the question, if it is worth to use the QDir overhead here, to express that the path is a Dir while all the rest of the API just takes QStrings anyway. I don't consider this as a good trade off.

Reply:
I don't think that a minor performance decrease matter in a test case. More important is readability and type safety of the test case code - especially when the test case will be extended later.